lib/tasks/github_usage_tracker.rake

```diff
@@ -0,0 +1,6 @@
+namespace :github_usage_tracker do
+  desc "Record a current datapoint of GitHub API usage"
+  task :record_datapoint => :environment do
+    GithubUsageTracker.record_datapoint
+  end
```

Comment:
We can also consolidate it back in to a normal queue with less frequency later if we want; for our current priorities I'd like to keep a fair number of as-accurate-as-possible observations.